mods/cnc/chrome/lobby-servers.yaml

```diff
@@ -85,10 +85,12 @@ Container@LOBBY_SERVERS_BIN:
 					X: 2
 					EnableChildMouseOver: True
 					Children:
-						Label@TITLE:
+						LabelWithTooltip@TITLE:
 							X: 5
-							Width: 245
+							Width: 225
```

Comment:
Why change this? The original value should already have the correct margins wrt the password/auth icons.

Reply:
This is part of the TESTCASE